ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Worker/XmlToArrayTransformer.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Should we try incorporating existing workers, such as: https://github.com/transfer-framework/commons/blob/1.0/src/Transfer/Commons/Xml/Worker/Transformer/StringToSimpleXmlTransformer.php?